src/java.management/share/classes/sun/management/VMManagementImpl.java

```diff
@@ -255,8 +246,7 @@ private synchronized PerfInstrumentation getPerfInstrumentation() {
         }
 
         // construct PerfInstrumentation object
-        @SuppressWarnings("removal")
-        Perf perf =  AccessController.doPrivileged(new Perf.GetPerfAction());
+        Perf perf =  Perf.getPerf();
```

Comment:
```suggestion
        Perf perf = Perf.getPerf();
```

Reply:
ok yes, an existing double-space but we'll fix it 8-)